admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -400,7 +400,7 @@ var featuresCollection = (function() {
 
         if('' !== $(this).val()) {
           $.ajax({
-            url: $(this).attr('data-action').replace(/\/\d+((?=\?.*))?/, '/' + $(this).val()),
+            url: $(this).attr('data-action').replace(/\/\d+(?!.*\/\d+)((?=\?.*))?/, '/' + $(this).val()),
```

Comment:
Maybe this part could be factored into a method with a more semantic meaning?

```js
doSomethingThatShouldBeExplainedHere = function(foo, bar) {
  return foo.replace(/\/\d+(?!.*\/\d+)((?=\?.*))?/, '/' + bar)
}
```

Reply:
alright